src/features/cell-renderers/prettyBytesCellRenderer.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+import { BYTES_IN_MB } from './constants';
+import baseCellRenderer from './baseCellRenderer';
+import getSize from '../../utils/size';
+import type { CellRendererParams } from './flowTypes';
+
+const prettyBytesCellRenderer = (multiplier: number = BYTES_IN_MB) => (cellRendererParams: CellRendererParams) =>
+    baseCellRenderer(cellRendererParams, cellValue => getSize(Number(cellValue) * multiplier));
```

Comment:
We should have the prettyBytes expecting bytes as the number, or we can rename this cellRender to sizeCellRenderer so it doesn't tie with bytes.